src/app/Plans/components/Wizard/helpers.tsx

```diff
@@ -308,7 +308,9 @@ export const getVMTreePathInfo = (
         .map((node) => node.object) as ICommonTreeObject[]) || null;
   }
   return {
-    datacenter: hostTreeAncestors?.find((node) => node.kind === 'Datacenter')?.object || null,
+    datacenter:
+      hostTreeAncestors?.find((node) => node.kind === 'Datacenter' || node.kind === 'DataCenter')
```

Comment:
I wonder, could we just coerce these with `toLowerCase()`?

Reply:
hmm, good point. I do like how TypeScript enforces that both the strings I compare against here are part of the string union type for that field though. It's such a tiny little thing that it's probably fine.

Reply:
Good call @seanforyou23, while fixing that I realized there were 2 other places where I should have been checking it case-insensitively too. Pushed a commit.

Reply:
Sweet!